### DISCUSSION TITLE
Exclusive accordion

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#176] [State of HTML Features]
Group `<details>` elements so that only up to one in the group can be open at a time.

### Code sample

```html
<details open name="sidebar_panel" id="main_info">
	<summary>Main info</summary>
	<!-- controls -->
</details>
<details open name="sidebar_panel" id="style_settings">
	<summary>Style</summary>
	<!-- controls -->
</details>
```

### Specification URL:

https://github.com/whatwg/html/pull/9400

### Any other links?

- [TAG review](https://github.com/w3ctag/design-reviews/issues/866#issuecomment-1661272254)
- [Open UI Exploration](https://open-ui.org/components/accordion.explainer/)
- [Chrome Status](https://chromestatus.com/feature/6710427028815872)

### How stable is this feature?

Future (No implementations yet)

### Type of feature

HTML attribute

### What focus area does this feature fall under?

Interactivity

### Details

The `name` attribute groups creates groups of `<details>` elements, where opening one closes the others, allowing easier implementation of accordions.

**Comment by ssg** [comment]
Duplicate elements with the same ID's seem to be against the HTML spec. A fieldset like container seems to be a better way to approach this.

**Reply by LeaVerou** [reply]
They do not share the same id.

**Reply by ssg** [reply]
That's not what the example shows?

**Reply by LeaVerou** [reply]
Typo! Fixed now, thanks.
